submissions/peterIyiola/DOM/newsLetter.js

```diff
@@ -0,0 +1,26 @@
+
+        let sendNews = document.getElementById('newsField');
+        let newsLetter = document.getElementById('newsLetter');
+
+function validations(field, event) {
```

Comment:
Since you are validating just one field, no need to passing a parament `field` to it. Simply use the `newsLetter` directly into the function